terraform/modules/google-cloud/apps/elixir/network.tf

```
@@ -65,6 +65,19 @@ resource "google_compute_security_policy" "default" {
     }
   }
 
+  rule {
+    description = "block sanctioned countries"
+
+    action   = "deny(451)"
+    priority = "101"
+
+    match {
+      expr {
+        expression = "request.path.matches(\"/sign_up\") && origin.region_code in ('RU', 'BY', 'KP', 'IR', 'SY')"
```

We need these as well, and will need to block service access too (can do it in another PR after announcement is made):

```suggestion
        expression = "request.path.matches(\"/sign_up\") && origin.region_code in ('RU', 'BY', 'KP', 'IR', 'SY', 'CU', 'VE')"
```

Hmm do you know if regions are supported? Ideally we need to block these as well:

```
Crimea, Donetsk, Lubansk
```

---

@jamilbk I double checked and they do have special region codes for them: XC and XD for NLB, will test if they work on App Armor too (and duplicate rules for LB if not): https://cloud.google.com/armor/docs/network-edge-policies